mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -21,230 +29,280 @@ instances:
   - server: localhost
 
     ## @param user - string - required
-    ## Datadog Username created to connect to MySQL.
+    ## Datadog username created to connect to MySQL.
     #
-    user: datadog
+    user: <USER>
 
     ## @param pass - string - required
-    ## Password associated with the datadog user.
+    ## Password associated to the Datadog user.
     #
-    pass: <UNIQUEPASSWORD>
+    pass: <PASS>
 
-    ## @param port - integer - optional - default: 3306
+    ## @param port - number - optional - default: 3306
     ## Port to use when connecting to MySQL.
     #
     # port: 3306
 
     ## @param sock - string - optional
-    ## Set the sock parameter if you want to connect to MySQL using a Unix Socket.
+    ## Path to a Unix Domain Socket to use when connecting to MySQL (instead of a TCP socket).
     #
-    # sock: <SOCK_PATH>
+    # sock: <SOCK>
 
     ## @param defaults_file - string - optional
-    ## Enter the path of an alternate configuration mechanism file.
+    ## Path to an alternative configuration mechanism file.
     #
-    # defaults_file: <CONFIGURATION_FILE_PATH>
+    # defaults_file: <DEFAULTS_FILE>
 
-    ## @param connect_timeout - integer - optional - default: 10
-    ## Change the timeout time in second for the Agent queries.
+    ## @param connect_timeout - number - optional - default: 10
+    ## Maximum number of seconds to wait before timing out when connecting to MySQL.
     #
     # connect_timeout: 10
 
-    ## @param tags - list of key:value elements - optional
-    ## List of tags to attach to every metric, event and service check emitted by this integration.
-    ##
-    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    ## Use this section to configure a TLS connection between the Agent and MySQL.
     #
-    # tags:
-    #   - <KEY_1>:<VALUE_1>
-    #   - <KEY_2>:<VALUE_2>
+    ssl:
 
-    ## @param ssl - object - optional
-    ## Use this parameter to configure a SSL connection between the Agent and MySQL:
-    ## `key`: Path to your key file.
-    ## `cert`: path to your cert file.
-    ## `ca`: path to your ca file.
-    #
-    # ssl:
-    #   key: <KEY_FILE_PATH>
-    #   cert: <CERT_FILE_PATH>
-    #   ca: <CA_PATH_FILE>
```

Comment:
(nit) Similar to `custom_queries`, but at lesser extent. I think the old way is easier to use and more readable.